H2MLauncher.UI/Converters/InvertableBooleanToVisibilityConverter.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Globalization;
+using System.Windows;
+using System.Windows.Data;
+
+namespace H2MLauncher.UI.Converters
+{
+    [ValueConversion(typeof(bool), typeof(Visibility))]
+    public class InvertableBooleanToVisibilityConverter : IValueConverter
+    {
+        enum Parameters
+        {
+            Normal, Inverted
+        }
+
+        public object Convert(object value, Type targetType,
+                              object parameter, CultureInfo culture)
+        {
+            var boolValue = (bool)value;
```

Comment:
... using var but still call it boolValue.

Reply:
Hahaha isnt it obvious though in this case 😂 
`bool boolValue = (bool)notBoolValue` xD